golang/main.go

```diff
@@ -1,7 +1,29 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"math"
+)
 
+func Log(x float64) float64 {
```

Comment:
```suggestion
func Log5(x float64) float64 {
```
чтобы не перекрывать стандартный log